### DISCUSSION TITLE
请问有方便国内使用的预设表情的CDN地址吗？

### DISCUSSION THREAD

**Discussion by east4ming** [walinejs/waline#841] [Q&A] [answered]
https://github.com/walinejs/waline/blob/11dd310eeefe9372c68a602b0857c432a9939753/docs/src/guide/client/emoji.md

文档中的地址还是 cdn.jsdelivr.net
目前国内访问缓慢，请问有方便国内使用的预设表情的CDN地址吗？
谢谢！

**Comment by lizheming** [comment]
https://github.com/walinejs/waline/discussions/741 有类似讨论~

**Comment by Mister-Hope** [comment, marked as answer]
目前已经按照 `@waline/emoji` 发布了 npm 包，你可以自行找一个国内访问速度快的镜像。

**Reply by heganyuliang** [reply]
国内镜像似乎只能识别 `/:package@:version/:file` 的格式，比如 [https://github.elemecdn.com/@waline/emojis@1.0.1/alus](https://github.elemecdn.com/@waline/emojis@1.0.1/alus) 就会报错

**Reply by Mister-Hope** [reply]
访问的时候会自动访问info.json

**Reply by heganyuliang** [reply]
谢谢，直接使用确实是可行的
